apim/3.x/README.adoc

```diff
@@ -208,7 +263,39 @@ pfx) |`+null+`
 
 See
 https://github.com/helm/charts/tree/master/stable/elasticsearch[Elasticsearch]
-for detailed documentation on optional requirements helm chart.
+for detailed documentation on optional requirements Helm chart.
+
+
+=== 4. Install The Gravitee Helm Chart
+
+To install the chart from the Helm repository with the release name
+`+graviteeio-apim3x+`, run:
+
+[source,bash]
+----
+$ helm install --name graviteeio-apim3x graviteeio/apim3
```

Comment:
This command installs APIM.
If we want the configuration file created previously to be considered, we have to add the `-f values.yaml` parameter